tools/staticbuild/build_lib_cmake.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
```

Comment:
set -eo pipefail . ?


maybe better in python?

Reply:
This is alike `build_lib.sh` for the make build. Let's set `set -eo pipefail` in both scripts?

With respect to having a bash script: We can migrate to a package manager (conan, vcpk, ...) based solution in the future.